Preprocessing/Normalizer.ecl

```diff
@@ -0,0 +1,54 @@
+/*###################################################################################
+## HPCC SYSTEMS software Copyright (C) 2020 - 2021 HPCC Systems.  All rights reserved.
+##################################################################################### */
+
+IMPORT $.^ as ML_Core;
+IMPORT ML_Core.Types AS MTypes;
+
+/**
+ * Normalize the data with below options:
```

Comment:
I'm not sure why we would want to do this???  Please explain the use, or remove this module.